README.md

```diff
@@ -48,7 +48,7 @@ Please check out the [issue queue](https://github.com/EddieJaoudeCommunity/suppo
 - **What is a GitHub Organization?** Essentially it's a way to group many repositories in an entity called an organization. An organization also allows us to group people who contribute and are a part of the community. Some examples would be: [Netflix](https://github.com/Netflix), [Facebook](https://github.com/facebook) and of course [EddieHub](https://github.com/EddieHubCommunity).
 Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-organizations).
 - **How do I join the Github Organization?** Raise an issue on this repository asking to join ([invite me](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization)).
-- **How do I set the Organization to Public?** You can make the organization public by: 
+- **How do I set the Organization to Public?** Here's how to make the organization public: 
```

Comment:
Is this one really necessary? Rethink.

Reply:
@adityaraute The later sounds more direct. The listed items beneath would be able to start with needing to add "-ing"